ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -438,8 +451,8 @@ private static void sanitizeMobileSearchKeys(ObjectNode searchNode, Pattern patt
       if (match.matches()) {
         String prefix = match.group("prefix");
         String code = match.group("code");
-        String channel = match.group("channel");
-        boolean codeIsValid = ALLOWED_SEARCH_CODES.contains(code);
+        String channel = match.group("channel").toLowerCase();
```

Comment:
I'm not entirely sure it's appropriate to lowercase the channel value as the values are lowercased on the client side, and I don't think we'd necessarily want to allow through `TS` instead of `ts`.